pkgs/top-level/python-packages.nix

```diff
@@ -7890,6 +7890,8 @@ in {
 
   subdownloader = callPackage ../development/python-modules/subdownloader { };
 
+  subliminal = callPackage ../development/python-modules/subliminal { };
```

Comment:
it seem to be there in master

https://github.com/NixOS/nixpkgs/blob/master/pkgs/top-level/python-packages.nix#L8435